include/parser_interface.hpp

```diff
@@ -39,6 +39,21 @@ class ParserInterface
         return types::DbusVariantType();
     }
 
+    /**
+     * @brief API to write keyword's value on hardware.
+     *
+     * @param[in] i_paramsToWriteData - Data required to perform write.
+     *
+     * @return On success returns no.of bytes written on hardware, On failure
```

Comment:
> no.of

There should be space in between. Also, better use number instead.

Reply:
done